static/src/stylesheets/module/external/_from-content-api.scss

```diff
@@ -167,11 +167,26 @@
 .caption {
     @include fs-textSans(1);
     color: $neutral-2;
-}
 
-.caption--img {
-    @include mq(leftCol) {
-        padding-bottom: $gs-baseline * 2;
+    .inline-information {
+        margin-right: $gs-gutter/10;
+        margin-top: -$gs-baseline/4;
+        width: 14px;
+        height: 14px;
+        background-color: $neutral-2;
+
+        svg {
```

Comment:
Should try and use a class here instead of style the element. Should be a `.inline-information__svg` class I think